docs/docs/topics/device-identity.md

```diff
@@ -0,0 +1,82 @@
+### Authenticated Device Types
+
+Device identity is made possible through trusted execution environment (**TEE**) devices that provide a hardware-attested identity. The specific implementation of this general concept is different across the devices that provide device identity, but they generally fall into two categories: secure enclaves and cross-platform security keys.
+
+#### Secure Enclaves
+
+Also called platform or internal authenticators, a secure enclave is physically bound to a specific computing device.
+
+- TPM (Trusted Platform Module): These devices are usually built into a product's mainboard, or can be installed in devices with a TPM header, as shown [here][toms-hardware-tpm]. They include a small processor to carry out cryptographic functions on the device, instead of on the system's processor where it could be interfered with. Trust is usually derived from a private key or certificate signed by a trusted manufacture's certificate authority.
+
+- Mobile devices: Most newer Apple and Android devices include a [Secure Enclave][apple-enclave] or [Hardware-backed Keystore][android-keystore]
+
+#### Hardware Security Keys
+
+Also known as cross-platform or roaming authenticators, these are authentication devices which can move with the user across different computers.
+
+- FIDO U2F: This [open standard][fido-spec] is used by many products like Yubico's [Yubikey][yubikey-products] and Google's [Titan Security Key](https://support.google.com/titansecuritykey/answer/9115487?hl=en). They usually secure a private key used to decrypt information signed by an accessible public key.
+
+::: tip Note
+The nature of cross-platform keys mean they are not associated with a single end-user device. Pomerium policies can be written to allow these keys, or specified to only accept secure enclaves.
+:::
+
+## Implement Device Identity with Pomerium
+
+Pomerium supports policies that use device identity since version [0.16.0](/docs/upgrading.md#policy-for-device-identity). We use the [Web Authentication][webauthn-api] (**WebAuthN**) API to bring authentication and authorization based on device identity into your security framework. It enables users to register their device ID, and admins to require a trusted device before accessing one, several, or all services.
+
+You can review our implementation of the WebAuthn specification [on GitHub](https://github.com/pomerium/webauthn).
+
+To get started, review the following pages:
+
+- [Pomerium Policy Language](/docs/topcics/ppl.md) to learn how to build policies that use device ID.
+- [Enroll a Device](/guides/enroll-device.md) to teach end-users how to enroll devices on Pomerium.
+
+
+## Looking Ahead: Device Posture
+
+Even if they only use the right device, what happens when a exploit is discovered in their OS or browser? How do you restrict access until the exploit is patched? Expanded iterations of device identity methods account for this using **device posture**, sometimes referred to as device state.
```

Comment:
```suggestion
Even if access is restricted to known devices, what happens when a user is found to have a vulnerable OS or browser version? How can an administrator ensure their network is not exposed, and that the user's system is promptly patched and remediated?  As secure enclave technologies evolve, **device posture** -- which is sometimes referred to as device state -- will play an increasingly important role in not only authorization decisions but also in helping to quickly remediate vulnerable corporate devices. 
```

Who is they? An administrator? A user? I think we should say a user. 

> Expanded iterations of device identity methods account for this using **device posture**, sometimes referred to as device state

What's odd is we go on to call it device state in the next paragraph.

Reply:
Agreed on all points. This is another case of content moved/shuffled around that needed better editing for it's new place in the overall doc.

Reply:
This addition, if agreeable, would make me :+1: on removing the following paragraph.
```suggestion
Even if access is restricted to known devices, what happens when a user is found to have a vulnerable OS or browser version? How can an administrator ensure their network is not exposed, and that the user's system is promptly patched and remediated?  As secure enclave technologies evolve, **device posture** -- which is sometimes referred to as device state -- will play an increasingly important role in not only authorization decisions but also in helping to quickly remediate vulnerable corporate devices. Device posture is a more complex superset of device identity, with more information about the device and software being used to generate the resulting identifier.
```